diff -ura naemon-livestatus-1.4.4.orig/src/AndingFilter.h naemon-livestatus-1.4.4.new/src/AndingFilter.h
--- naemon-livestatus-1.4.4.orig/src/AndingFilter.h	2025-09-03 11:32:41.303412986 +0200
+++ naemon-livestatus-1.4.4.new/src/AndingFilter.h	2025-09-03 11:59:00.575214291 +0200
@@ -8,6 +8,7 @@
 #include "config.h"
 
 #include "Filter.h"
+#include <ctime>
 #include <deque>
 
 class AndingFilter : public Filter
diff -ura naemon-livestatus-1.4.4.orig/src/Filter.h naemon-livestatus-1.4.4.new/src/Filter.h
--- naemon-livestatus-1.4.4.orig/src/Filter.h	2025-09-03 11:32:41.305183315 +0200
+++ naemon-livestatus-1.4.4.new/src/Filter.h	2025-09-03 11:59:11.015105218 +0200
@@ -7,6 +7,7 @@
 
 #include "config.h"
 
+#include <ctime>
 #include <vector>
 #include <string>
 #include <stdint.h>
diff -ura naemon-livestatus-1.4.4.orig/src/TimeColumn.h naemon-livestatus-1.4.4.new/src/TimeColumn.h
--- naemon-livestatus-1.4.4.orig/src/TimeColumn.h	2025-09-03 11:32:41.310183373 +0200
+++ naemon-livestatus-1.4.4.new/src/TimeColumn.h	2025-09-03 12:04:36.515680358 +0200
@@ -10,6 +10,7 @@
 #include <stdint.h>
 
 #include "Column.h"
+#include <ctime>
 
 class TimeColumn : public Column
 {
